ui/.npmrc

```diff
@@ -0,0 +1,3 @@
+@fortawesome:registry=https://npm.fontawesome.com/
```

Comment:
This line was one of two in gs://all-of-us-workbench-test-credentials/.npmrc that we copied to our environment. One of those lines was the authentication token needed for this package. I've moved that secret into its own file so the rest of this non-secret file could be version-controlled.